### ISSUE TITLE
Tracker popup looks weird

### ISSUE THREAD

**Issue by marcinjahn** [rafaelmardojai/firefox-gnome-theme#460]
A moment ago I saw this popup:

![image](https://user-images.githubusercontent.com/10273406/196028857-1d015af4-275e-4e3f-b116-bdda8338f68e.png)

I have never seen it before, maybe it's a new thing from Firefox. Anyway, it looks weird, probably because of the theme.

**Comment by rafaelmardojai** [comment]
Yeah, I probably have seen it before, but haven't time to make it appear again 😅.
